docs/100-upgrade/migration-guides/3.1-3.2.mdx

```diff
@@ -88,3 +88,193 @@ AND
 (order.comment contains "Bar"
   OR order.comment contains "456")
 ```
+
+### ContributionMode
```

Comment:
**nitpick:** This isn't directly related to the Contribution mode. That's something that should be done for everyone.

**suggestion:** rename the section and add an introduction sentence explaining it a bit

```suggestion
### Remix entrypoints

In this release, we added new features that required to hook into your Remix application entrypoints created with the initial skeleton.

You must update these files in your application, as detailed below. 
```

Reply:
Section added first, renamed and added an introduction sentence.